hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/transport/server/ratis/ContainerStateMachine.java

```diff
@@ -265,6 +268,23 @@ public ContainerStateMachine(HddsDatanodeService hddsDatanodeService, RaftGroupI
 
   }
 
+  private void validatePeers(RaftServer server, RaftGroupId id) throws IOException {
+    if (this.peersValidated.get()) {
+      return;
+    }
+    RaftPeerId selfId = server.getId();
+    Collection<RaftPeer> peers = server.getDivision(id).getGroup().getPeers();
+    // If peers list is empty then it means  Ratis hasn't created any raft--meta file containing the last applied
+    // transaction. Then the peer list can be only validated on apply transaction.
```

Comment:
Peer list cannot be empty here.  Remove the check.

Reply:
done